File: ImageSlider/app/src/main/java/alifyz/com/imageslider/MainActivity.java
package alifyz.com.imageslider;

import android.support.v4.view.ViewPager;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    ViewPager mViewPager;
    ImagePageAdapter mAdapter;
    int[] mResourceId;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        mResourceId = new int[] {
                R.drawable.image1,
                R.drawable.image2,
                R.drawable.image3,
                R.drawable.image4,
                R.drawable.image5,
                R.drawable.image6};

        mViewPager = (ViewPager) findViewById(R.id.viewpager);
        mAdapter = new ImagePageAdapter(this, mResourceId);
        mViewPager.setAdapter(mAdapter);

    }
}

File: Journaler/app/src/main/java/alifyz/com/journaler/MainActivity.kt
package alifyz.com.journaler

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Toast

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun toastMe(view : View) {
        val myToast = Toast.makeText(this, "test", Toast.LENGTH_LONG);
        myToast.show();
    }
}
